### PR TITLE
chore(CHANGELOG): update to v4.7.0

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to #2157

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,43 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `CHORE`: capture used Camunda Forms in telemetry events ([#2188](https://github.com/camunda/camunda-modeler/issues/2188))
+
+### BPMN
+
+* `FEAT`: support creating, opening, deploying and starting Zeebe BPMN diagrams ([#2029](https://github.com/camunda/camunda-modeler/issues/2029))
+* `FEAT`: support non-default element templates for root elements ([#2121](https://github.com/camunda/camunda-modeler/issues/2121))
+* `FEAT`: support BPMN Errors on external service tasks ([#2070](https://github.com/camunda/camunda-modeler/issues/2070))
+* `FEAT`: introduce JSON Schema versioning via `$schema` property ([#2083](https://github.com/camunda/camunda-modeler/issues/2083))
+* `FEAT`: validate element templates against JSON Schema ([#2159](https://github.com/camunda/camunda-modeler/issues/2159))
+* `FEAT`: introduce status bar ([#2175](https://github.com/camunda/camunda-modeler/issues/2175))
```
Comment (barmac) [inline CHANGELOG.md:24]: Status bar is a general feature.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,43 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `FEAT`: introduce engine profile overlay ([#2187](https://github.com/camunda/camunda-modeler/issues/2187))
```
Comment (barmac) [inline CHANGELOG.md:25]: Same as above.

File: CHANGELOG.md (hunk 1 above)
Comment (pinussilvestrus) [reply CHANGELOG.md:24]: Ups, yeah sorry...

**Comment by barmac** [comment]
giphy

**Review by barmac — APPROVED** [review]

**Pull request merged**